### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by Brianna-Flo**
## Description

<what this pull request is doing>

- cleanup codebase comments and formatting

<what will be done in later PRs and not included here>

## Milestones
<the milestones or stories/features that this works towards>

- quality of code

## Resources
<links to tutorials, code snippets, inspirations>
<if you took or translated specific code from the internet, please call it out here!>

## Test Plan
<insert images or gifs of feature>
<otherwise, say how code was tested if not UI facing>
<any edge cases you might have specifically tested>

### CODE REVIEW

File: frontend/src/utils/utils.ts [hunk 1]
```diff
@@ -107,10 +108,10 @@ const validateInput = (formData: AuthFormData) => {
     return { type: "error", text: "Email and password are required" };
   }
 
-  if (formData.password.length < PASS) {
+  if (formData.password.length < MIN_PASSWORD_LENGTH) {
     return {
       type: "error",
-      text: "Password must be at least 8 characters long",
+      text: `Password must be at least ${MIN_PASSWORD_LENGTH} characters long`,
```
Comment (radhikadhomse) [inline frontend/src/utils/utils.ts:114]: nice, thanks for pulling it out intto a const

**Review by radhikadhomse — APPROVED** [review]
